### ISSUE TITLE
SelectMen && DropdownMenu cause a `react-dom` warning in client side console.

### ISSUE THREAD

**Issue by nonissue** [bumbag/bumbag-ui#31]
- `SelectMenu` and `DropdownMenu` (this could possibly affect other components), cause react dom to emit a warning message in the browser console. 

Sample: 

```js
[Error] Warning: Received `false` for a non-boolean attribute `wrap`.

If you want to write it to the DOM, pass a string instead: wrap="false" or wrap={value.toString()}.

If you used to conditionally omit it with wrap={condition && value}, pass wrap={condition ? value : undefined} instead.
    in button (created by ForwardRef(Button))
    in ForwardRef(Button)
    in ForwardRef(Button)
    in Unknown (created by Button)
    in Button (at DropdownTest.js:16)
    in div (created by ForwardRef(Box))
    in ForwardRef(Box)
    in Unknown (created by DropdownMenu)
    in DropdownMenu (at DropdownTest.js:6)
    in DropdownTest (at pages/index.js:17)
    in div (created by ForwardRef(Box))
    in div (created by ForwardRef(Box))
    in ForwardRef(Box)
    in Unknown (created by PageContent.Wrapper)
    in PageContent.Wrapper (created by ForwardRef(Box))
    in ForwardRef(Box)
    in Unknown (created by PageContent)
    in PageContent (at pages/index.js:9)
    in Index (at _app.js:6)
    in div (created by ForwardRef(Box))
    in ForwardRef(Box)
    in Unknown (created by Box)
    in Box (created by Provider)
    in PageProvider (created by Provider)
    in ToastProvider (created by Provider)
    in IdProvider (created by Provider)
    in SystemProvider (created by Provider)
    in unstable_IdProvider (created by Provider)
    in Provider (created by Provider)
    in div (created by ColorModeProvider)
    in ColorModeProvider (created by Provider)
    in ThemeProvider (created by Provider)
    in Provider (at _app.js:5)
    in MyApp
    in ErrorBoundary (created by ReactDevOverlay)
    in ReactDevOverlay (created by Container)
    in Container (created by AppContainer)
    in AppContainer
    in Root
```

I'm using nextjs, but the playroom example for these components also fails to render, so I suspect it's not limited to next projects.

To reproduce, just import SelectMenu from bumbag in a react file, and load the page with dev console open.

**Expected behavior**

**Device information (please complete the following information):**

-   Device: Macbook Pro, 2016
-   OS, version: macOS, 10.15.6
-   Browser, version: Safari 14 Preview (also checked firefox).

### Simple implementation of SelectMenu

```js
const Tags = () => {
  const [value, setValue] = useState();

  return (
    <SelectMenu
      label='Tags'
      isMultiSelect
      hasTags
      onChange={setValue}
      options={[
        { key: 1, label: 'Apples', value: 'apples' },
        { key: 2, label: 'Bananas', value: 'bananas' },
        { key: 3, label: 'Oranges', value: 'oranges' },
        { key: 4, label: 'Mangos', value: 'mangos' },
      ]}
      placeholder='Select a fruit...'
      value={value}
    />
  );
};
```

### Screenshot of react component tree, with subcomponents of DropdownMenu possessing the `wrap` prop.

<img width="941" alt="apw-cat local -  2020-08-12 at 2 23 08 AM" src="https://user-images.githubusercontent.com/5218392/89992645-c9cf9c00-dc42-11ea-8e53-c0d996bd6ae8.png">

My best guess it needs to be omitted from the components props when the component is being implemented.

Thanks!

**Comment by makkabi** [comment]
Same with the Autocomplete component, I tried it with the code snippets from the Docs and Create React App on a Mac in different Browsers, so nothing specific at all.

**Comment by jxom** [comment]
Fixed on `1.0.11`
